v2.1/demo-data-replication.md

```diff
@@ -137,78 +137,83 @@ Open the Admin UI at <a href="http://localhost:8080" data-proofer-ignore>http://
 
 ## Step 5. Increase the replication factor
 
-As you just saw, CockroachDB replicates data 3 times by default. Now, in the terminal you used for the built-in SQL shell or in a new terminal, use the [`cockroach zone`](configure-replication-zones.html) command change the cluster's `.default` replication factor to 5:
+As you just saw, CockroachDB replicates data 3 times by default. Now, in the terminal you used for the built-in SQL shell or in a new terminal, use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) statement to change the cluster's `.default` replication factor to 5:
 
 {% include copy-clipboard.html %}
 ~~~ shell
-$ echo 'num_replicas: 5' | cockroach zone set .default --insecure --host=localhost:26257 -f -
+$ cockroach sql --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=6;" --insecure --host=localhost:26257
```

Comment:
This changes the number of replicas to 6, not 5 as stated in the above paragraph.  Related: If I remember correctly we recommend having odd numbers of replicas across the cluster.  The idea is that if you have 6 replicas of system ranges and have a "split brain" scenario with 3 sets each on either side of a network partition, the cluster can't correct the fault since there will not be a majority.  This might be a separate issue from your change anyhow, but we should document this somewhere IMO. For reference we will apparently ignore setting `num_replicas` to an even number as of 2.1 - see https://github.com/cockroachdb/cockroach/issues/30435